ansible_rulebook/app.py

```diff
@@ -107,6 +101,12 @@ async def run(parsed_args: argparse.Namespace) -> None:
 
     validate_actions(startup_args)
     validate_variables(startup_args)
+    startup_args.variables = decrypted_context(startup_args.variables)
```

Comment:
Do we have to move this block from the worker condiiton?

Reply:
@bzwei Yes so that we have the decrypted_context in one place. The default env vars is just an empty dictionary which is not used for CLI's but only used when the data is coming from EDA Server